nav2-usability/scenario-code/skipping-stacks/skipping_stacks_vrouter.dart

```diff
@@ -0,0 +1,187 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Skipping stacks example
+/// Done using VRouter
+
+import 'package:flutter/material.dart';
+import 'package:vrouter/vrouter.dart';
+
+void main() {
+  runApp(BooksApp());
+}
+
+class Book {
+  final String title;
+  final Author author;
+
+  Book(this.title, this.author);
+}
+
+class Author {
+  String name;
+
+  Author(this.name);
+}
+
+class AppState extends ChangeNotifier {
+  final List<Book> books = [
+    Book('Stranger in a Strange Land', Author('Robert A. Heinlein')),
+    Book('Foundation', Author('Isaac Asimov')),
+    Book('Fahrenheit 451', Author('Ray Bradbury')),
+  ];
+
+  List<Author> get authors => [...books.map((book) => book.author)];
+}
+
+class BooksApp extends StatelessWidget {
+  final AppState _appState = AppState();
+
+  @override
+  Widget build(BuildContext context) {
+    return VRouter(
+      routes: [
+        // Books
+        VWidget(
+          path: '/',
+          widget: BooksListScreen(books: _appState.books),
+          stackedRoutes: [
+            VWidget(
+              path: r'book/:bookId(\d+)',
+              widget: Builder(
+                builder: (context) => BookDetailsScreen(
+                  book: _appState.books[
+                  int.parse(context.vRouter.pathParameters['bookId']!)],
+                ),
+              ),
+            ),
+          ],
+        ),
+
+        // Authors
+        VWidget(
+          path: '/authors',
+          widget: AuthorsListScreen(authors: _appState.authors),
+          stackedRoutes: [
+            VWidget(
+              path: r'/author/:bookId(\d+)',
+              widget: Builder(
+                builder: (context) => AuthorDetailsScreen(
+                  author: _appState
+                      .books[
+                  int.parse(context.vRouter.pathParameters['bookId']!)]
+                      .author,
+                ),
+              ),
```

Comment:
Same here, this might be more readable:

```suggestion
              widget: Builder(
                builder: (context) {
                  final bookId =
                      int.parse(context.vRouter.pathParameters['bookId']!);
                  return AuthorDetailsScreen(
                    author: _appState.books[bookId].author,
                  );
                },
              ),
```